### ISSUE TITLE
Remove optional parameters from example models

### ISSUE THREAD

**Issue by tsmbland** [EnergySystemsModellingLab/MUSE_OS#499]
In #495 I went through the documentation and marked the parameters which are optional. Usually these parameters are only needed if using certain agent objectives.

To keep the example models and tutorials simple, I suggest that we remove these parameters and only include the parameters that are required for these models to run. 

The main reason to do this is that most users start with the default model and add to it to build their own models. If a parameter (such as the CO2 emissions factor) is included in the default model, and the user doesn't understand that it's optional, they make go through considerable (and wasted) effort to find the appropriate value for any new component that they add, or be unnecessarily intimidated by the complexity of the model.

**Comment by Jin-Jun-Mei** [comment]
I agree that keep the tutorial simple will help users to build the model faster.

But I think there are some parameters that are necessary . For instance, you mentioned the "CO2 emissions factor." In my opinion, for an energy system model (that focused on low-carbon transitions), this is an important parameter. I would actually suggest making the "CO2 emissions factor" a non-optional parameter for the model (MUSE).

/Jinxi

**Comment by tsmbland** [comment]
Yeah, I'm not sure what the original intention of the `CommodityEmissionFactor_CO2` parameter was, but it isn't used anywhere, in any circumstances, and the `CommOut` file has all the information MUSE needs about emissions.
